wgsl/index.bs

```diff
@@ -12233,8 +12233,8 @@ The zero values are as follows:
 * The zero value for an *C*-column *R*-row matrix of type *T* is the matrix of those dimensions filled with the zero value for *T*.
 * The zero value for a [=constructible=] *N*-element array with element type *E* is an array of *N* elements of the zero value for *E*.
 * The zero value for a [=constructible=] structure type *S* is the structure value *S* with zero-valued members.
-* The zero value of an [=AbstractInt=] is 0.
-* The zero value of an [=AbstractFloat=] is 0.0.
+* The zero value of an [=AbstractInt=] is `0`
+* The zero value of an [=AbstractFloat=] is `0.0`
```

Comment:
Is the omission of trailing dots intentional? Thank you

Reply:
Yes.
I don't want readers to mistake this for part of the syntax of the token.